signac/contrib/job.py

```diff
@@ -449,6 +449,16 @@ def _init(self, force=False):
         """
         fn_manifest = os.path.join(self._wd, self.FN_MANIFEST)
 
+        # Attempt early exit if the manifest exists and is valid
+        try:
+            self._check_manifest()
+        except Exception:
+            # Any exception means this method cannot exit early
+            pass
```

Comment:
Instead of a `pass`, might I suggest to remove the complete rest of the function block here? Then you can also remove lines 458..461. It will create a larger diff, but the logic is clearer.

Reply:
I think I did what you wanted. Please check in 459632ec3e859d63ce8eabf4d324cceb0767ceb8.